Python/omevv/v1/scripts/GetOMIVVProfileDetails.py

```diff
@@ -1,4 +1,5 @@
 import argparse
+import sys
```

Comment:
WHy do we need this sys?

Reply:
@pavithra-mahadev : It was used in the script to handle the exception, so i just retained it.

print("get_bearer_token: Unexpected error:", sys.exc_info()[0])